app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Please rename template file to **blog_post_list.phtml** or **latest_blog_posts.phtml**

Reply:
fixed